src/parser/jobs/mnk/Fists.js

```diff
@@ -50,6 +54,20 @@ export default class Fists extends Module {
 	}
 
 	_onComplete() {
+		this.checklist.add(new Rule({
+			name: 'Keep Fists of Fire up',
+			description: <Fragment>
+				Fists of Fire is a low effort 5% buff to all your damage.
+			</Fragment>,
+			requirements: [
+				new Requirement({
+					name: <Fragment><ActionLink {...ACTIONS.FISTS_OF_FIRE} /> uptime</Fragment>,
+					percent: () => this.getBuffUptimePercent(STATUSES.FISTS_OF_FIRE.id),
+				}),
+			],
+			target: 90,
```

Comment:
I am by no means an expert at MNK, but I feel like this could maybe be higher? I know RoE is a thing but in my understanding, you usually don't want to linger in FoE after you get the stack refresh. Not a blocker, just something to think about.

Reply:
This number kinda came out of just averaging diff FFLogs but it's actually really variable. For a player doing a basic rotation, it should be easily 95+%, but for players that do frequent Tornado Kicks and use RoW for recovery, the number ends up as low as 60%.